modules/visual_script/visual_script.cpp

```diff
@@ -131,8 +132,8 @@ VisualScriptNode::TypeGuess VisualScriptNode::guess_output_type(TypeGuess *p_inp
 	return tg;
 }
 
-Ref<VisualScript> VisualScriptNode::get_visual_script() const {
-	return script_used;
+Ref<Resource> VisualScriptNode::get_container() const {
```

Comment:
Can this be a more specific class than resource?

Reply:
Hmmm.. it will be hard as atm, as the base b/w VisualScript and VisualScriptModule are not common class, though we can create a class that extends from Script that can be used as the base for both. The problem is that VisualScriptModule isn't handled as a Script atm, so we will have to add all the virtual functions required by Script base as well, on top of all VisualScript functions that are needed in the Base Class.